src/pc/lua/smlua_ffi.c

```
@@ -0,0 +1,54 @@
+        lc = lc->next;
+    }
+
+    void* bound_func;
+    ffi_closure* closure = ffi_closure_alloc(sizeof(ffi_closure), &bound_func);
+    if (closure == NULL) {
+        LOG_ERROR("Failed to allocate ffi_closure");
+        return NULL;
+    }
+
+    lc = malloc(sizeof(struct LuaClosure));
+    lc->closure = closure;
```

Cppcheck (2.20.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)